gateway/src/main/java/io/zeebe/gateway/impl/job/RoundRobinActivateJobsHandler.java

```diff
@@ -66,13 +68,15 @@ public void activateJobs(
       final int maxJobsToActivate,
       final String type,
       final Consumer<ActivateJobsResponse> onResponse,
+      final Consumer<Throwable> onError,
```

Comment:
Optional: Maybe rename to `onErrorConsumer`/`onErrorHandler` (dito for` onNext/Completed`)